### DISCUSSION TITLE
Don't understand instllation process

### DISCUSSION THREAD

**Discussion by hik200** [Frederisk/Wikitext-Extension-Gadget#6] [Q&A]
You say that need just enter line into settings but i am unable to find any text area in Settings->Appearance

If installing as a gadget I need to insert files into the extension folder.
can you give compiled version?
and where need enter the script?

**Comment by Frederisk** [comment]
![image](https://user-images.githubusercontent.com/29837738/210070862-e7175c1f-6062-4453-aca9-9ce9f776b035.png)

Decide where you want to add this gadget. click its **Custom JavaScript** hyperlink.

![image](https://user-images.githubusercontent.com/29837738/210071189-ce74f0ef-d795-4636-b4d7-88120d62dc1e.png)

Edit and append the code.

**Comment by evoludolab** [comment]
Your gadget appears to be a perfect fit for curing my wiki-editing frustrations. 

Unfortunately I can't seem to figure out how to install the gadget. In my Preferences > Appearance I see the list of skins but no links to Custom CSS or Custom Javascript, just Preview. The shared entries are missing as well

I have a MediaWiki 1.40 installation. Is this a feature that was added later? 

Can't wait to try your gadget - Wikitext on VSCode is ready :-) Your help is much appreciated.

**Reply by Frederisk** [reply]
Check [this](https://www.mediawiki.org/wiki/Manual:Interface/JavaScript) to make sure you have the functionality enabled correctly on your site.

Generally speaking, you need to turn [`$wgAllowUserJs`](https://m.mediawiki.org/wiki/Manual:$wgAllowUserJs) on if you want to customize content for a single user.

**Reply by evoludolab** [reply]
Thanks for your quick response! With your help the gadget is now installed. Looking forward to test drive everything. 🎉
